CHANGELOG.rst

```diff
@@ -44,6 +44,11 @@ Changes
 * Exceptions are now all accessible in the ``exceptions`` module and are descended from the ``GirderBaseException`` class.
 * Require npm 5.2+ (with npm 5.6+ strongly recommended) to build the web client
 
+Web Client
+^^^^^^^^^^
+* Use ``href`` properties for navigation links in addition to JavaScript onclick events. (`#2489 <https://github.com/girder/girder/pull/2489>`_)
+* Change instances of ``.g-server-config`` to ``a.g-server-config`` to enable adding of ``href`` properties to those links
```

Comment:
This selector won't actually work, since it's not an element that is both an `a` and a `.g-server-config`. Still, I think this provides more than adequate notice for anyone that may care about this particular change in the admin console page (which I doubt anyone would).

Reply:
Oops... I pushed one last doc fix. Once the tests pass I'll merge this. Thanks Brian!

Reply:
Oh, @brianhelba, I guess you'll have to re-review after the tests pass.